app/models/work.rb

```diff
@@ -0,0 +1,22 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+  has_many :users, through: :votes
+
+  def self.spotlight
+    works = Work.all
+    max = 0
+    works.each do |work|
+      max = work.votes.count if work.votes.count > max
+    end
```

Comment:
You can just use `max` here.  It accepts a block.

```suggestion
    max = works.max { |work| work.votes.count }
```